modules/infrastructure-moving-router.adoc

```diff
@@ -14,10 +14,10 @@ is displayed to a worker node.
 
 .Procedure
 
-. View the `clusteringress` Custom Resource for the router Operator:
+. View the `IngressController` Custom Resource for the router Operator:
 +
 ----
-$ oc get clusteringress default -n openshift-ingress-operator -o yaml <1>
+$ oc get ingresscontroller default -n openshift-ingress-operator -o yaml <1>
 ----
 <1> The router is managed by an Operator that is named
 `openshift-ingress-operator`, and its Pod is in the `openshift-ingress-operator`
```

Comment:
This is not quite right.  It is true that the router (which is being renamed to "ingress controller"—Brandi has another PR for this renaming) is managed by the Ingress Operator, and the operator's pod is in the `openshift-ingress-operator` project, but I wouldn't say that the operator is named `openshift-ingress-operator`.  Anyway, I don't think that the operator's name or namespace are relevant in describing this command, which is getting the ingress controller named `default`, which is in the `openshift-ingress-operator` namespace.